lib/vintage_net_wizard/application.ex

```diff
@@ -3,16 +3,23 @@ defmodule VintageNetWizard.Application do
 
   use Application
 
+  alias VintageNetWizard.{Backend, Web.Endpoint}
+
   @spec start(Application.start_type(), any()) :: {:error, any} | {:ok, pid()}
   def start(_type, _args) do
     backend = Application.get_env(:vintage_net_wizard, :backend, VintageNetWizard.Backend.Default)
 
     children = [
-      {VintageNetWizard.Web.Endpoint, []},
-      {VintageNetWizard.Backend, backend}
+      {Endpoint, []},
+      {Backend, backend}
     ]
 
     opts = [strategy: :one_for_one, name: VintageNetWizard.Supervisor]
     Supervisor.start_link(children, opts)
   end
+
+  def start_phase(:wizard_startup, _start_type, _phase_args) do
+    unless Backend.configured?(), do: VintageNetWizard.run_wizard()
```

Comment:
Doesn't this break handling of `Backend` GenServer crashes? I.e. If the there's a restart and the Backend isn't configured, then is the wizard not started?

Also, I don't understand how this change is related to loading previous configurations.

Reply:
By this point the backend has been started and configured successfully. If backend were to have bad configuration and crash, it would happen while be started by the supervisor before getting to the start phase.

The relation question is a can of worms - maybe better to have a quick call in the morning. TL;DR there are lots of circulation deps between using the backend and server during startup and restarting the wizard (like with a button press)

Reply:
Yeah, I'll take a look more at this tomorrow.